extensions/cornerstone/src/utils/dicomLoaderService.js

```diff
@@ -188,6 +189,9 @@ class DicomLoaderService {
       );
     } else if (!someInvalidStrings(wadoUri)) {
       return fetchIt(wadoUri, { headers: authorizationHeaders });
+    } else if (!someInvalidStrings(instance?.url)) {
+      const uri = instance.url.replace(/^dicomweb:/, '');
```

Comment:
Can you add a `GRADIENT_README.md` file on what a "segmentation file is suppose to look like? I'm a bit confused why this is needed since seg files should also be dicomweb:// files